tests/ColorDistanceLineTraceTest.cpp

```diff
@@ -0,0 +1,286 @@
+/**
+ * @file   ColorDistanceLineTraceTest.cpp
+ * @brief  ColorDistanceLineTraceクラスのテスト
+ * @author miyahara046
+ */
+
+#include "ColorDistanceLineTrace.h"
+#include <gtest/gtest.h>
+#include <gtest/internal/gtest-port.h>
+
+using namespace std;
+
+namespace etrobocon2025_test {
+  // 最初3回の色取得で連続して指定色を取得するテストケース
+  TEST(ColorDistanceLineTraceTest, runToGetFirst)
```

Comment:
テスト名はアッパーキャメルケースでお願いします🙇他のところも。
```suggestion
  TEST(ColorDistanceLineTraceTest, RunToGetFirst)
```